PCampReview.py

```diff
@@ -158,15 +155,80 @@ def infoPopup(self,message):
     messageBox = qt.QMessageBox()
     messageBox.information(None, 'Slicer mpMRI review', message)
 
+  def setupIcons(self):
```

Comment:
No, I don't think there is minimum resolution, or default size of icons for tab bars.